resources/js/components/updater/Release.vue

```diff
@@ -18,6 +18,17 @@
         <div class="card-body">
             <div v-html="release.body"></div>
         </div>
+
+        <confirmation-modal
+            v-if="confirmationPrompt"
+            :title="packageName"
+            :bodyText="__('Are you sure you want to :type to :version?', { type: confirmationPrompt.type, version: confirmationPrompt.version })"
```

Comment:
We'll need two separate translations for update vs downgrade.

We can't pass an English word into a translation.

Reply:
Ah good catch! How'd you propose I do that, as in, use different translation keys depending on the release type?

Reply:
```html
:body-text="confirmationText"
```
```js
computed: {
  confirmationText() {
    const attrs = { version: this.confirmationPrompt.version };
    return this.confirmationPrompt.type === 'downgrade'
      ? __('Are you sure you want to downgrade to :version?', attrs)
      : __('Are you sure you want to update to :version?', attrs);
  }
}
```

I know that could be cleaned up further using string interpolation or a single `__()`, but by doing it this way, the `php translator` tool will be able to pick up those two explicit strings.

Reply:
Didn't think of a `computed` property but that's exactly the kind of thing they're built for 😅